apps/internal/base/base.go

```diff
@@ -302,3 +306,30 @@ func convertStrUnixToUTCTime(unixTime string) (time.Time, error) {
 	}
 	return time.Unix(timeInt, 0).UTC(), nil
 }
+
+func getKeyFromRequest(authParams authority.AuthParams) string {
```

Comment:
So this looks like something that should just be attached to authority.AuthParams.  Renamed to CacheKey().

Reply:
Resolved in https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/184/commits/71dea81ddef28df17b4741bef3e8643d75e7ccd9